### ISSUE TITLE
Wrong autocomplete dropdown placement

### ISSUE THREAD

**Issue by MrDach** [ant-design-blazor/ant-design-blazor#1813]
### Describe the bug

The dropdown for the lookup patterns is appended to the root element instead of the parent of the dropdown component and thus placed in the top left corner of the pages.

### Steps to reproduce (please include code)

This bug can be seen in your documentation: [](https://ant.design/components/auto-complete/)
![image](https://user-images.githubusercontent.com/22943367/128025303-d97139c0-6c0c-4d3f-b759-a38828728735.png)

**Comment by speyck** [comment]
I also had some problems with the AutoComplete DropDown "placement". The DropDown was placed over any item in the UI. I had a navbar that is sticky at the top and when you open an AutoComplete and scroll down, the DropDown is over the navbar. 

Maybe the devs can take a look at AutoComplete and overwork it a bit..

**Comment by anranruye** [comment]
@MrDach thanks for contacting us.

I can't see the dropdown anywhere, even not at the top-left of the page. There must be something wrong with this demo or the autocomplete component.

@speyck https://antblazor.com/en-US/components/autocomplete#components-autocomplete-demo-popup-container can this solve your problem?

**Comment by speyck** [comment]
@anranruye are you referring to the `fix popup container`? If I try that the popup just floats in the middle of the page.

**Comment by MrDach** [comment]
Any suggestions on a quick fix for this? I'm guessing there's some javascript somewhere that is placing the DOM element incorrectly.

**Comment by anranruye** [comment]
@speyck I think you can give the navbar a larger z-index.

**Comment by anranruye** [comment]
@MrDach Yes, we changed the javascript recently, which might be the cause.

**Comment by MrDach** [comment]
I've created the following quickfix for now:

When firing the OnPanelVisibleChange event I will execute the following javascript function if the panel is visible:

```
window.FixAutoCompleteLocation = function () {
    const dropdownWrapper = document.createElement("div");
    dropdownWrapper.className = "dropdownWrapper";
    dropdownWrapper.setAttribute("style", "position: relative;");
    dropdownWrapper.appendChild(document.querySelector('.ant-select-dropdown'));

    document.querySelector('.ant-input-search').appendChild(dropdownWrapper);
}
```

After applying some flexbox styling to fix the width it now it looks like this :)

![image](https://user-images.githubusercontent.com/22943367/128356167-a3c6881a-d3d1-4207-95bb-d9f1a2026457.png)

**Comment by ElderJames** [comment]
This dropdown can not be closed.

![auto-complete-cant-close](https://user-images.githubusercontent.com/7550366/130450194-a1cbfd9f-36d5-4bc7-a5f7-5dd3e2ae91ed.gif)
